### ISSUE TITLE
[Dev](User Profile)Merge private profile changes to the public profile

### ISSUE THREAD

**Issue by ethanzlevin** [dev-launchers/dev-launchers-platform#916]
Jude knows what to do

**Comment by judeamos** [comment]
Once #895 is done, this story will automatically be done as well.

**Comment by judeamos** [comment]
I have completed #895. There are additional changes I must to for this story to be fully completed. This works alongside #895.

**Comment by judeamos** [comment]
We will revisit this in a new story for the user profile revamp.
